### ISSUE TITLE
train error

### ISSUE THREAD

**Issue by FBEMPSS** [hologerry/Attr2Font#13]
![2020-11-05 15-38-57屏幕截图](https://user-images.githubusercontent.com/39507189/98211735-74730480-1f7d-11eb-9777-6bc852b1de1a.png)
when input image size is 64 in style encoder，it will be transformed to a tensor which size is (1, 165, 1 ,1)
and it will be fed into residual black , which has a conv(kernel = 3) and generate an error.

**Comment by Kiode** [comment]
I meet this error also, have you solve this problem?

**Comment by sq75985** [comment]
I also met the same problem.
Then it was solved by reinstalling a different version of Pytorch.
I’m not sure, may be you can check your Pytorch version, especially the CUDA .

**Comment by ueoo** [comment]
Since the older instance norm uses batch norm as the backend, there are maybe some issues when the batch size is 1.

**Comment by RongXueE** [comment]
> I also met the same problem. Then it was solved by reinstalling a different version of Pytorch. I’m not sure, may be you can check your Pytorch version, especially the CUDA .

May I ask u about your server environment configuration? I also encountered this problem, so sad

**Comment by Pesciolini** [comment]
> > I also met the same problem. Then it was solved by reinstalling a different version of Pytorch. I’m not sure, may be you can check your Pytorch version, especially the CUDA .
> 
> May I ask u about your server environment configuration? I also encountered this problem, so sad

Are u solve this problem? I also confused about it, so sad.

**Comment by Pesciolini** [comment]
> > > I also met the same problem. Then it was solved by reinstalling a different version of Pytorch. I’m not sure, may be you can check your Pytorch version, especially the CUDA .
> > 
> > 
> > May I ask u about your server environment configuration? I also encountered this problem, so sad
> 
> Are u solve this problem? I also confused about it, so sad.

Use torch 1.4 & python 3.6 help me solve this problem
